app/content/survey-manual.md

```diff
@@ -0,0 +1,261 @@
+<script async
+  subscriptions-control="manual"
+  type="application/javascript"
+  src="https://news.google.com/swg/js/v1/swg{{#env.SWG_OVERRIDE}}-{{.}}{{/env.SWG_OVERRIDE}}.js">
+</script>
+
+
+# Survey prompts
+
+This page both demonstrates how RRM:E survey sign-up works, as well as documents
+how to implement the prompts.
+
+## Test the manual survey invocation
+
+<div id="prompts"></div>
+
+## Survey Prompt Overview
+
+A publisher can configure one or more surveys for manual invocation by using an
+initialized `swg.js` instance. In order to use this feature:
+
+1.  The publisher will create a survey in the Publisher Center content access section 
+    for their publication.
+1.  The publisher will fetch a valid prompt using the `configurationId`, and then 
+    display it.
+1.  When configuring the prompt display code, a publisher will provide a callback
+    that will be used to store the responses from the newsletter signup, and 
+    acknowledge to Google that the email address has been saved.
+
+
+### Prompt Configuration Examples
+
+<ul class="flexible-list">
+  <!-- <li class="flexible-list-item">
+    <img src="/img/newsletter-configurations_0000s_0004_standard.png">
+    <pre>
+    <code class="hljs language-json">
+{
+  "publicationId": "asdf-1234",
+  "name": "Daily Bugle"
+}
+    </code>
+    </pre>    
+  </li> -->
+  <li class="flexible-list-item">
+    <img src="/img/survey.png">
+    <pre>
+    <code class="hljs language-json">
+{
+  "name": "Basic survey",
+  "configurationId": "49c12712-9750-4571-8c67-96722561c13a"
+}
+    </code>
+    </pre> 
+  </li>
+    <li class="flexible-list-item">
+    <img src="/img/survey-multi-page.png">
+    <pre>
+    <code class="hljs language-json">
+{
+  "name": "Multi-page survey",
+  "configurationId": "e98a2efb-d009-43c9-99ef-dda11c8c5a7f"
+}
+    </code>
+    </pre> 
+  </li>
+</ul>
+
+After creating a configuration, the Publisher Center page will provide a `configurationId` in response for each
+newsletter configuration. These ids will then be used in subsequent javascript api calls.
+
+```json
+[
+  {
+    name: 'Basic survey',
+    configurationId: '49c12712-9750-4571-8c67-96722561c13a',
+  },
+  {
+    name: 'Multi-page survey',
+    configurationId: 'e98a2efb-d009-43c9-99ef-dda11c8c5a7f',
+  },
+]
+```
+
+### Invoke Survey Prompts
+
+To configure survey prompts, `swg.js` must first be configured on the page.
+These examples show using the initialization of the library in `manual` mode, but the
+APIs are also available in automatic mode. 
+
+### Get the prompt instance to display
+
+To invoke a survey prompt, a publisher must use the `configurationId` provided by
+Google in response to submitting a prompt configuration. Publishers use the
```

Comment:
Publishers get the available configurationId by using the ... (?)

Reply:
`subscriptions.getAvailableInterventions()`? There's some example code right under here

Reply:
I added a note re: Publisher Center page.